lib/src/models/image/image.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:dart_mappable/dart_mappable.dart';
+import 'package:meta/meta.dart';
+
+import '../../api.dart';
+import '../../get_image_url.dart' as get_image_url;
+import '../../hosts.dart';
+import 'cover.dart';
+import 'image_thumbnail.dart';
+import 'image_type.dart';
+
+part 'image.mapper.dart';
+
+
+/// Image.
+@immutable
+@MappableClass(
+  includeCustomMappers: [
+    ImageTypeMapper,
+  ],
+)
+class Image with ImageMappable {
+  const Image({
+    required this.id,
+    required this.media,
+    required this.type,
+    this.width,
+    this.height,
+  });
+
+  static final fromMap = ImageMapper.fromMap;
+  static final fromJson = ImageMapper.fromJson;
+  static final fromValue = ImageMapper.container.fromValue<Image>;
```

Comment:
Why there we have fromValue, but other models misses it?

Reply:
What that method even does? Make docs for these `from`s. (For all of them, you can use regex if you want)